pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java

```diff
@@ -354,6 +377,8 @@ private PulsarWorkerService createPulsarFunctionWorker(ServiceConfiguration conf
         workerConfig.setAuthenticationEnabled(true);
         workerConfig.setAuthorizationEnabled(true);
 
+        workerConfig.setConnectorsDirectory(Files.createTempDirectory("test").toFile().getAbsolutePath());
```

Comment:
what about naming the directory "tempconnectorsdir" ?

Reply:
sure